src/renderer/components/settings/Information.tsx

```diff
@@ -47,9 +44,12 @@ export class LanguageSettings extends React.Component<Props, States> {
 
     public async componentDidMount() {
         const { locale } = this.props;
-
+        let folderPath: string = path.join((global as any).__dirname, "assets/md/information");
         try {
-            let fileContent = await promisify(readFile)(`src/resources/information/${locale}.md`, {encoding: "utf8"});
+            if (__PACKAGING__ === "0") {
+                folderPath = path.join(process.cwd(), "dist/assets/md/information");
```

Comment:
The string `assets/md/information` is duplicated, instead it should be defined once, for example:

```javascript
const infoFolderRelativePath = "assets/md/information";
// ...
let folderPath: string = path.join((global as any).__dirname, infoFolderRelativePath);
// ...
folderPath = path.join(process.cwd(), "dist", infoFolderRelativePath);
```

Other than that, this now looks clean to merge :)

Reply:
Done